### PR TITLE
homeTask7

### PR DESCRIPTION
**Pull request by MaksimKuznets**

### CODE REVIEW

File: homeTask_7/homeTask_7/Person.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        public string Name{get; }
+        public string LastName { get; }
+
+        public Person(string name, string lastName)
+        {
+            this.Name = name;
+            this.LastName = lastName;
+        }
+
+        public Person()
```
Comment (EgorAntonovich) [inline homeTask_7/homeTask_7/Person.cs]: Do we need this?

File: homeTask_7/homeTask_7/Employee.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace homeTask_7
+{
+    abstract class Employee: Person
+    {
+        public Employee(string name, string lastName): base(name, lastName)
+        {
+
+        }
+        public abstract string GetAction();
```
Comment (EgorAntonovich) [inline homeTask_7/homeTask_7/Employee.cs]: Why you aren't use this?

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: homeTask_7/homeTask_7/Person.cs (hunk 1 above)
Comment (MaksimKuznets) [reply homeTask_7/homeTask_7/Person.cs]: > Do we need this?
Do you mean the узьен constructor?

File: homeTask_7/homeTask_7/Employee.cs (hunk 2 above)
Comment (MaksimKuznets) [reply homeTask_7/homeTask_7/Employee.cs]: > Why you aren't use this?

So, if you are about GetAction() method, then I really don't use it. I don't remember why i implemented it. I will remove that.

File: homeTask_7/homeTask_7/Person.cs (hunk 1 above)
Comment (MaksimKuznets) [reply homeTask_7/homeTask_7/Person.cs]: @EgorAntonovich  I will remove implementation of empty constructor

**Review by EgorAntonovich — APPROVED** [review]
Looks good

**Pull request merged**